binary-decode.go

```diff
@@ -630,9 +631,12 @@ func (cdc *Codec) decodeReflectBinaryStruct(bz []byte, info *TypeInfo, rv reflec
 	}
 	_n := 0 // nolint: ineffassign
 
+	// Keep incrementing the last fieldnum. Later, when we support custom fieldtag numbers, we need to store the last
```

Comment:
should be declared in the case statement in the switch statement?

Reply:
Thx, done.